x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/BertJapaneseTokenizationUpdate.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.ml.inference.trainedmodel;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.core.Nullable;
+import org.elasticsearch.xcontent.ConstructingObjectParser;
+import org.elasticsearch.xcontent.ParseField;
+import org.elasticsearch.xcontent.XContentParser;
+import org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class BertJapaneseTokenizationUpdate extends AbstractTokenizationUpdate {
```

Comment:
I think there is potential to reduce some of the code duplication by reusing the `BertTokenizationUpdate`. But that is not necessary for this PR.

Reply:
If you mean `BertJapaneseTokenizationUpdate` should be inherited from `BertTokenizationUpdate`, then I have a concern to do that. Assume that there is a instance of BertJapaneseTokenizationUpdate called `ja`, then that `ja` object is also  an instance of BertTokenizationUpdate. So if we write a code for TokenizationUpdate like below in the future, then I'm afraid it potentially become problematic.

https://github.com/elastic/elasticsearch/blob/8e7d4b075092869dbae8e9a9511c13adbb62c30f/x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/NlpTokenizer.java#L390-L398

https://github.com/daixque/elasticsearch/blob/5890fca9214c2c823a5c513fb0f42c602ab8ce7a/x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/NlpTokenizer.java#L391-L402